hackathon_site/dashboard/frontend/src/pages/Cart/Cart.test.tsx

```diff
@@ -93,4 +97,23 @@ describe("Cart Page", () => {
         // Test for quantity
         expect(getByText(0)).toBeInTheDocument();
     });
+
+    test("removeFromCart button", async () => {
+        const store = makeStoreWithEntities({
+            hardware: mockHardware,
+            cartItems: mockCartItems,
+        });
+
+        const { getByTestId, queryByTestId, getByText } = render(<Cart />, {
+            store,
+        });
+
+        const removeButton = getByTestId(mockCartItems[2].hardware_id.toString());
+        fireEvent.click(removeButton);
+
+        await waitFor(() => {
+            expect(queryByTestId(mockHardware[2].name)).not.toBeInTheDocument();
```

Comment:
You should use queryByText here instead if test id because you're trying to see id the hardware name is not on the page anymore